Source/AssistantV1/Assistant.swift

```diff
@@ -211,8 +214,6 @@ public class Assistant {
         if let headers = headers {
             headerParameters.merge(headers) { (_, new) in new }
         }
-        let sdkHeaders = Shared.getSDKHeaders(serviceName: serviceName, serviceVersion: serviceVersion, methodName: "message")
-        headerParameters.merge(sdkHeaders) { (_, new) in new }
```

Comment:
Whoops!  This is another failure on my part. You need to specify something in the config to get the generator to include this code, and I should have documented this.